README.md

```diff
@@ -14,95 +17,82 @@ or set of validation rules (best practices).
 [![][error prone img]][error prone]
 [![][pitest img]][pitest]
 [![][checker framework img]][checker framework]
-
 [![][dependabot img]][dependabot]
-
 [![][mavenbadge img]][mavenbadge]
 [![][sonar img]][sonar]
-
 [![][release notes/version img]][release notes/version]
-
 [![][closed issues img]][closed issues]
 [![][link check img]][link check]
-
 [![][milestone img]][milestone]
 
-Contributors chat: [![][matrix_con img]][matrix_con]
-
-The latest release version can be found at
-[GitHub releases](https://github.com/checkstyle/checkstyle/releases/)
-or at [Maven repo](https://repo1.maven.org/maven2/com/puppycrawl/tools/checkstyle/).
-
-Each-commit builds of maven artifacts can be found at
-[Maven Snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/com/puppycrawl/tools/checkstyle/).
-
-Documentation is available in HTML format, see https://checkstyle.org/checks.html.
-
-## Build instructions and Contribution
-
-[Build instructions](https://checkstyle.org/contributing.html#Build)
+## Table of Contents
+1. [Quick Start](#quick-start)
+2. [Contributing](#contributing)
+3. [Feedback and Support](#feedback-and-support)
+4. [Project Reports](#project-reports)
+5. [Sponsor Checkstyle](#sponsor-checkstyle)
+6. [Licensing](#licensing)
 
-[Setup IDE for development](https://checkstyle.org/beginning_development.html)
+## Quick Start
 
-[Explanation on how to create your own module](https://checkstyle.org/extending.html)
+- Download our [Latest Release](https://github.com/checkstyle/checkstyle/releases/) from GitHub.
+- Add Checkstyle to your build from [Maven Central](https://search.maven.org/search?q=g:%22com.puppycrawl.tools%22%20AND%20a:%22checkstyle%22).
+- Read our [Documentation](https://checkstyle.org/checks.html) for usage and configuration.
+- Get the latest snapshot from the [Maven Snapshot Repository](https://oss.sonatype.org/content/repositories/snapshots/com/puppycrawl/tools/checkstyle/).
 
-[Verification of code quality](https://checkstyle.org/contributing.html#Quality_matters)
+## Contributing
 
-[Sending Pull Request](https://checkstyle.org/contributing.html#Submitting_your_contribution)
+Thanks for your interest in contributing to CheckStyle! Please see the
+[Contribution Guidelines](https://github.com/checkstyle/checkstyle/blob/master/.github/CONTRIBUTING.md)
+for information on how to contribute to the project. This includes creating issues, submitting pull
+requests, and setting up your development environment. **If you are a potential Google Summer
+of Code (GSoC) participant, this is a great place to start.**
```

Comment:
So we don't want to tell them about the new GSOC MD page?

Reply:
readme should have nothing on gsoc